podcasts/PlaylistSelectionViewController.swift

```diff
@@ -0,0 +1,116 @@
+import PocketCastsDataModel
+import PocketCastsUtils
+import UIKit
+
+class PlaylistSelectionViewController: PCViewController, UITableViewDelegate, UITableViewDataSource {
+    private static let filterAutoDownloadCell = "FilterDownloadCell"
```

Comment:
Should this be renamed to `playlistAutoDownloadCell` or just `autoDownloadCell`?

```suggestion
    private static let playlistAutoDownloadCell = "PlaylistDownloadCell"
```

Reply:
Thought to rename it in another PR but I just changed it